test/parallel_api/iterator/permutation_iterator.pass.cpp

```diff
@@ -56,7 +56,7 @@ struct is_able_to_modify_src_data_in_test : ::std::true_type { };
 #if TEST_DPCPP_BACKEND_PRESENT
 template <typename ExecutionPolicy>
 struct is_able_to_modify_src_data_in_test<ExecutionPolicy, perm_it_index_tags::counting>
```

Comment:
I believe all specializations of 
struct is_able_to_modify_src_data_in_test<ExecutionPolicy, perm_it_index_tags::yyyyy>
derived from `std::true_type` not required anymore.

Reply:
Agreed.  I think we actually may be able to enable all these cases, and perhaps even remove this check...  Investigating it now.